src/d3a/models/strategy/__init__.py

```diff
@@ -230,6 +234,45 @@ def __init__(self):
 
     parameters = None
 
+    def rotate_profile(self, profile_type: InputProfileTypes, profile,
```

Comment:
I think that both methods can be moved to the global_objects.profile_db_connection. That way profile handling will be centralized to one class and the strategy code will remain clean from it.

Reply:
Here I am not convinced. This method is also dealing with rotating the profiles that are not in the DB. Only for the profiles stored in the DB, `global_objects.profile_db_connection.get_profile_from_db_buffer(profile_uuid)` is called. Happy to move it somewhere else, thought. But where?

Reply:
I think the best approach is to create another class that wraps up this functionality. Therefore, instead of having only the ProfileDBConnectionHandler class, you could create another wrapper class, that also lives in the global_objects:
`class ProfileHandler:
    def __init__(self): 
        self.db = ProfileDBConnectionHandler()
`
That way you will have a single interface for profile management, that internally uses either the db connection or the profiles that are not in the DB.